libraries/networking/src/SentPacketHistory.cpp

```diff
@@ -45,6 +48,10 @@ const QByteArray* SentPacketHistory::getPacket(uint16_t sequenceNumber) const {
     if (seqDiff < 0) {
         seqDiff += UINT16_RANGE;
     }
-
-    return _sentPackets.get(seqDiff);
+    auto packet = _sentPackets.get(seqDiff);
+    if (packet) {
+        return packet->get();
+    } else {
+        return nullptr;
+    }
```

Comment:
I believe the previous implementation was fine (and shorter), unique_ptr::get will return nullptr if it doesn't have a managed object